src/pages/Assets/Overview/YourSynths/YourSynths.tsx

```diff
@@ -80,6 +83,29 @@ export const YourSynths: FC<YourSynthsProps> = memo(
 			setSelectedSynth(null);
 		};
 
+		const handleAddToMetamask = async (currencyKey: string) => {
+			try {
+				const tokenDetails = await getCurrencyDetails(currencyKey);
+				const wasAdded = await window?.ethereum?.request({
+					method: 'wallet_watchAsset',
+					params: {
+						type: 'ERC20',
+						options: {
+							address: tokenDetails.tokenAddress,
+							symbol: tokenDetails.tokenSymbol,
```

Comment:
this one can just be `currencyKey`

Reply:
not sure we can drop the async/await on the 

` await window?.ethereum?.request({})`

bc in the ethereum docs it is said to be a promise :) correct me if I'm wrong

https://docs.metamask.io/guide/ethereum-provider.html#ethereum-request-args